### ISSUE TITLE
wsl.exe --help option garbles output when piped to more

### ISSUE THREAD

**Issue by henrik-jensen** [microsoft/WSL#4456]
* Your Windows build number:  (Type `ver` at a Windows Command Prompt)

Microsoft Windows [Version 10.0.18362.295]

* What you're doing and what's happening: 

Start cmd.exe prompt
```
c:\> wsl --help | more
```
Result - every char is postfixed with newline it seams:

![wsl_--help_vs_more](https://user-images.githubusercontent.com/9413242/64048548-4e302880-cb72-11e9-83bb-cb7df77f000f.png)

* What's wrong / what should be happening instead:

Readable (`more`) paged output expected.

PS!
This is not a console issue. It's the `wsl.exe --help` option only, outputting extra (newline?) chars.
e.g. `c:\> wsl ls --help | more` works fine.
PS!!
Tested with both `c:\> chcp 850` (default on my system) and `c:\> chcp 65001`, unicode output. Same result.

**Comment by henrik-jensen** [comment]
Brute workaround (working on my system win 10 1903 danish ):

```shell
c:\>wsl --help | wsl tr -d '\000\015' | more
```
but of course that seems quite fragile though.

The format that 'wsl --help' spits out seems to be some bastard kind of utf16LE but where every '\r\n' becomes '\r\r\n'.
So the above effectively creates a UTF8 with unix line ends output, but will only work on ascii text.

**Comment by therealkenc** [comment]
Interesting report. Yep there is an extra `\r` in there alright. For the giggles I created a UTF16LE file with VS Code (I never touch the stuff personally) and `System32\more.com` does eat it correctly. So `wsl.exe's` output does appear to be malformed.

Alternative work-around below. I set my Windows system locale to Chinese with the new fangled Windows Terminal v0.4.2382.0 [preview](https://github.com/microsoft/terminal/releases).

```
C:>wsl --help | wsl iconv -c -f utf16 -t utf8 | more
```

Seems to take believe it or not:

![image](https://user-images.githubusercontent.com/7049700/64060564-796f4380-cb83-11e9-8b4e-a2ba2247ada3.png)

No word on whether that text makes any sense (I believe it does not) but it does match the non-more output.

**Comment by henrik-jensen** [comment]
@therealkenc - Much better and robust workaround, also works in the standard (old) console (The one used when calling cmd.exe, what's its name? ).

**Comment by therealkenc** [comment]
Windows Console (vs newspeak Windows Terminal). Yes those words are synonyms but they had to call it something.

I found with wide characters like Chinese the (scare quote) "old" Windows Console mangled some glyphs. I just tried Windows Terminal on a lark. Western-ish charsets will possibly be fine in both. Can't speak to Danish, lolz.

**Comment by NotTheDr01ds** [comment]
Really the same root cause as #4607, for which a workaround is available starting with release [0.64.0](https://github.com/microsoft/WSL/releases/tag/0.64.0).  I've confirmed that setting `$env:WSL_UTF8=1` (or the equivalent in other shells) causes `wsl --help | more` to display correctly.

For those who aren't able to upgrade to Windows 11 at this time, the workaround in [this comment]($env:WSL_UTF8=1) by falloutphil should also work.
